### PR TITLE
Desktop version

### PR DESCRIPTION
**Pull request by codewithed**
Completed the following tasks for the desktop version:

Added a media query to implement mobile-first responsive design
Added appropriate images for certain elements
Added a nav-bar for the header section
Removed hamburger icon for desktop-version
Added appropriate styling in accordance with the template provided.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -33,10 +38,10 @@ <h1>I'm Edmund<br> Glad to see you</h1>
         </section>
         <section class="works grid-container">
```
Comment (codecaiine) [inline index.html:39]: - [ ] Your project section has white color as `background-color` . Please use a `background-color: grey` to make it looks similar to the Figma template :smile: 

### Figma template
Screenshot from 2022-07-15 11-01-27

### Your work
Screenshot from 2022-07-15 11-01-02

- [ ] Please make sure that your project's picture and the project's description, switch places with each  as you can see on Figma template :+1:

File: index.html [hunk 2]
```diff
@@ -121,46 +126,55 @@ <h2 class="project-title">Multi-Post Stories</h2>
         </section>
 
         <section class="myself">
```
Comment (codecaiine) [inline index.html:128]: - [ ] Please make sure that the left part of your about myself takes 40% and the right one the remaining 60% of the page's width. Kindly alter it so it looks similar to the Figma design :+1: 

- [ ] Your experience description after about myself title looks bigger compared to Figma template, please make that you are using the correct  `font-size`. :+1: 

- [ ] Please make sure that your 3 rows have drop-down menus for the Languages, Frameworks, and Skills. :smile: 

### Figma template
Screenshot from 2022-07-15 11-14-05

### Your project
Screenshot from 2022-07-15 11-12-25

File: index.html [hunk 3]
```diff
@@ -121,46 +126,55 @@ <h2 class="project-title">Multi-Post Stories</h2>
...
                 <ul class="icon">
-                    <li class="icon"><a> <img src="images/twitter-icon.svg" alt="twitter"/> </a></li>
-                    <li class="icon"><a> <img src="images/medium-icon.svg" alt="medium"/> </a></li>
-                    <li class="icon"><a> <img src="images/github-icon.svg" alt="github"/> </a></li>
-                    <li class="icon"><a><img src="images/angellist-icon.svg" alt="angellist"></a></li>
+                    <li><a><img src="images/twitter-icon.svg" alt="twitter"/></a></li>
+                    <li><a><img src="images/medium-icon.svg" alt="medium"/></a></li>
+                    <li><a><img src="images/github-icon.svg" alt="github"/></a></li>
+                    <li><a><img src="images/angellist-icon.svg" alt="angellist"></a></li>
                 </ul>
```
Comment (codecaiine) [inline index.html:142]: - [ ] Your social media links seem to have different styling compared to the Figma template, kindly change it so it looks similar to the Figma template :+1: 

### Your work
Screenshot from 2022-07-15 11-27-03

### Figma template
Screenshot from 2022-07-15 11-26-44

**Review by codecaiine — CHANGES_REQUESTED** [review]
Hi @codewithed , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!
<FEEL FREE TO CHANGE THIS MESSAGE TO YOUR PERSONAL WORDS>

## Highlights
- Good use of Gitflow :heavy_check_mark: 
- Commits are meaningful :heavy_check_mark: 
- Clean code :heavy_check_mark:  

## Required Changes ♻️

- [x] You are doing well however make sure to stick to the design as much as possible (e.g., font, colors, images, text, margins) using :+1: 

_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me (@codecaiine) in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by MeqdamAlqudah — APPROVED** [review]
Hi @codewithed , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**